webpack.config.js

```diff
@@ -62,7 +63,8 @@ module.exports = {
       "selectors": absolute("./content-src/selectors"),
       "lib": absolute("./content-src/lib"),
       "strings": absolute("./strings"),
-      "test": absolute("./content-test")
+      "test": absolute("./content-test"),
+      "l20n": absolute("./node_modules/l20n/dist/compat/web/l20n.js")
```

Comment:
I had to use the `compat` variant which is writtern in es5 because uglifyjs doesn't like some of the es6 syntax from `dist/bundle/web/l20n.js`.